### PR TITLE
Fixed deprecations

### PR DESCRIPTION
**Pull request by core23**
I am targetting this branch, because this is a patch.

## Changelog

```markdown
### Changed
 - The `sonata-project/block-bundle` is an optional dependency

### Fixed
- Fixed BlockBundle deprecation messages
- Fixed request service deprecation messages
- Fixed old center html tag
```

## Subject

Fixed deprecated methods calls and html tags.

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -23,6 +23,7 @@
         "symfony/framework-bundle": "^2.3",
         "symfony/templating": "^2.3",
         "sonata-project/admin-bundle": "^3.4",
+        "sonata-project/block-bundle": "^3.2",
```
Comment (greg0ire) [inline composer.json]: Optional dependency too? Maybe move it to require-dev?

**Comment by core23** [comment]
Fixed it

**Comment by greg0ire** [comment]
Travis is KO

**Comment by core23** [comment]
Travis will be green after merging this: https://github.com/sonata-project/SonataNewsBundle/pull/361

**Comment by OskarStark** [comment]
#361 is merged @core23, please rebase

**Comment by greg0ire** [comment]
I think there is no need to rebase.

**Comment by greg0ire** [comment]
Build triggered again

**Comment by OskarStark** [comment]
Thank you @core23

**Pull request merged**